app/services/claims/fetch_eligible_misc_fee_types.rb

```diff
@@ -41,7 +41,11 @@ def agfs_scheme_scope
     end
 
     def eligible_lgfs_misc_fee_types
-      filter_trial_only_types(lgfs_fee_types_by_claim_type, apply_clar_rep_order_filter? || apply_trial_fee_filter?)
+      filter_trial_only_types(lgfs_fee_types_by_claim_type, apply_filter_for_lgfs?)
+    end
+
+    def apply_filter_for_lgfs?
+      apply_clar_rep_order_filter? || apply_trial_fee_filter? || apply_transfer_guilty_plea_filter?
```

Comment:
could use Enumerable#any?, but just a preference
`[apply_clar_rep_order_filter?, apply_trial_fee_filter?, apply_transfer_guilty_plea_filter?].any?

Reply:
I prefer the former.

There is a (very small) performance benefit of `test_one || test_two || test_three` as it will stop evaluating when the first `true` is returned. Using `Enumerable#any` will evaluate the entire array.